packages/portal-proto/src/features/user-flow/workflow/registeredApps.tsx

```diff
@@ -228,6 +229,18 @@ export const REGISTERED_APPS = [
     noDataTooltip:
       "Current cohort does not have gene expression data available for visualization.",
   },
+  {
+    name: "Cohort Level MAF",
+    // TODO: replace with a MAF icon
+    icon: <CohortLevelMAFIcon aria-hidden="true" />,
+    tags: ["ssm"],
+    hasDemo: false,
+    countsField: "sequenceReadCaseCount",
+    description: "Combine and download cohort level MAF data",
```

Comment:
extreme nitpick on my end. can you add a period "." at the end of the description (for consistency with the other descriptions)?

Reply:
I'll add the period in my next commit when the `countsField` is changed to something more appropriate.